API/projectClass/multiTypeTable.m

```diff
@@ -57,17 +65,25 @@
         end
         
         function obj = setValue(obj, row, col, value)
-            % Change the value of a given parameter in the table. The row
-            % and column of the parameter can both be specified by either
-            % name or index. The expected input is three values: row,
-            % column, value
+            % Change the value of a given parameter in the table. 
             %
-            % multiTable.setValue(1, 1, 'origin');
-            % First parameter needs to be either a row name or number
+            % The row and column of the parameter can both be specified by either
```

Comment:
Try "Both the row and column of the parameter can be specified by either name or index" here. I think it reads a bit cleaner.